sefaz/cuf/uf.go

```diff
@@ -0,0 +1,68 @@
+package cuf
+
+import "errors"
+
+// Cuf stands for "Codigo da unidade federativa" and it
+// is strongly associated with Stakeholder.
+type Cuf string
```

Comment:
How about CUf instead of Cuf? (case)

Reply:
Initially a thought on CUF, all caps, because its acronym, but kept the original case from where I pulled the code from. Do you think CUF would be idiomatic enough?